index.go

```diff
@@ -20,9 +37,18 @@ func rootHandler(
 }
 
 func main() {
+    addr, err := determineListenAddress()
```

Comment:
why not `port` instead of `addr`?

Reply:
its an address cause it has a preceding ":" and its a string

Reply:
after some googling looks like something like `:80` is referred to as the port, and `80` is the port number.